src/org/zaproxy/zap/extension/bugTracker/PopupSemiAutoIssue.java

```diff
@@ -0,0 +1,140 @@
+ * 
+ * Copyright 2010 psiinon@gmail.com
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); 
+ * you may not use this file except in compliance with the License. 
+ * You may obtain a copy of the License at 
+ * 
+ *   http://www.apache.org/licenses/LICENSE-2.0 
+ *   
+ * Unless required by applicable law or agreed to in writing, software 
+ * distributed under the License is distributed on an "AS IS" BASIS, 
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. 
+ * See the License for the specific language governing permissions and 
+ * limitations under the License. 
+ */
+package org.zaproxy.zap.extension.bugTracker;
+
+import java.awt.Component;
+
+import javax.swing.JTree;
+import javax.swing.tree.DefaultMutableTreeNode;
+import javax.swing.tree.TreePath;
+
+import org.parosproxy.paros.Constant;
+import org.parosproxy.paros.control.Control;
+import org.parosproxy.paros.core.scanner.Alert;
+import org.parosproxy.paros.extension.ExtensionPopupMenuItem;
+import org.parosproxy.paros.extension.history.ExtensionHistory;
+import org.zaproxy.zap.extension.alert.ExtensionAlert;
+import org.parosproxy.paros.view.View;
+import java.awt.Dimension;
+import org.zaproxy.zap.extension.alert.AlertPanel;
+import org.zaproxy.zap.view.StandardFieldsDialog;
+
+public class PopupSemiAutoIssue extends ExtensionPopupMenuItem {
+
+	private static final long serialVersionUID = 1L;
+
+	private ExtensionAlert extension = null;
+    private ExtensionBugTracker extBT = null;
+
+	private ExtensionHistory extHist = null; 
+    private RaiseSemiAutoIssueDialog raiseSemiAutoIssueDialog = null;
+
+    public PopupSemiAutoIssue(ExtensionBugTracker ext) {
+        super();
+        this.extBT = ext;
+ 		initialize();
+    }
+
+    public PopupSemiAutoIssue(ExtensionBugTracker ext, String label) {
+        super(label);
+        this.extBT = ext;
+        initialize();
+    }
+
+    private void showRaiseSemiAutoIssueDialog(Alert[] alerts) {
+        if (raiseSemiAutoIssueDialog == null) {
+            this.extBT.alerts = alerts;
+            raiseSemiAutoIssueDialog = new RaiseSemiAutoIssueDialog(this.extBT, View.getSingleton()
+                    .getMainFrame(), new Dimension(500, 500));
+        } else if (raiseSemiAutoIssueDialog.isVisible()) {
+            bringToFront(raiseSemiAutoIssueDialog);
+            return;
+        }
+
+        raiseSemiAutoIssueDialog.setAlert(alerts);
+        raiseSemiAutoIssueDialog.setVisible(true);
+    }
+
+	private void initialize() {
+        this.setText(Constant.messages.getString("bugTracker.popup.issue.semi"));
+
+        this.addActionListener(new java.awt.event.ActionListener() { 
+
+        	@Override
+        	public void actionPerformed(java.awt.event.ActionEvent e) {
+        	    
+			    DefaultMutableTreeNode node = (DefaultMutableTreeNode) extension.getAlertPanel().getTreeAlert().getLastSelectedPathComponent();

```

Comment:
As per comment on the zaproxy PR, this class shouldnt access the extention.getAlertPanel() method directly. Instead it should extend a new abstract class which can be included in a temporary jar.